.github/workflows/integration_test.yaml

```diff
@@ -90,12 +90,13 @@ jobs:
       # `pro-airgapped-server` models share the same controller and therefore
       # can be integrated with each other over a cross-model relation.
       - name: Install LXD
-        uses: canonical/setup-lxd@v0.1.1
+        uses: canonical/setup-lxd@v0.1.2
         with:
           channel: 6.1/stable
+          group: lxd
```

Comment:
Also, I think you can get rid of this job because LXD is already installed in the runner. I remember I had this issue in #45, and removing this step worked at the end.

Reply:
Check out the changes in:
https://github.com/canonical/livepatch-k8s-operator/pull/45/files#diff-f61384d930ecca9b388b7d547b0c10b6c977f087b106b0747d64dad87a96daa0

You can also revert the changes to the `juju bootstrap` command.

Reply:
Also apply the same change to fix the license header issues:
https://github.com/canonical/livepatch-k8s-operator/pull/45/files#diff-f8a05717594bf148241760c18447442978fdc9db03a4aae93739529e0c124f8a